sos/report/plugins/networkmanager.py

```diff
@@ -110,6 +110,8 @@ def postproc(self):
                 self.do_file_sub(
                     "/etc/NetworkManager/system-connections/"+net_conf,
                     r"psk=(.*)", r"psk=***")
-
+                self.do_file_sub(
+                    "/etc/NetworkManager/system-connections/"+net_conf,
+                    r"password=(.*)", r"password=***")
```

Comment:
Can't this be merged with previous `do_file_sub` to `r"(psk|password)=(.*)", r"\1=***` ? (I hope I wrote the r.e. properly, writing it from top of my head..)

Reply:
@pmoravec Indeed that was the correct regex. Thank you for that!
I was sure there was a simpler/more correct way to do what I had in my first commit, that's why the PR was converted back to a draft.
With that change, it should be all ready for review.
Thanks again!